mantle/cmd/kola/kola.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	"github.com/coreos/coreos-assembler-schema/cosa"
+	cosa "github.com/coreos/coreos-assembler/pkg/builds"
```

Comment:
random aside: I'm wondering if we should not rename the import here.. not sure. I'm definitely don't have this all in my head.

Reply:
It's a matter of taste; I chose not to here to avoid inflating the diff more.  Probably in most code we wouldn't rename it though.